src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -1,27 +1,30 @@
-# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+# pylint: disable=missing-class-docstring, missing-function-docstring,protected-access,redefined-outer-name
 """
 Tests for forecast driver.
 """
 import datetime as dt
 import logging
-import os
 from pathlib import Path
 from unittest.mock import patch
 
 import pytest
+import yaml
 from pytest import fixture, raises
 
 from uwtools import scheduler
 from uwtools.config.core import NMLConfig, YAMLConfig
+from uwtools.config.j2template import J2Template
 from uwtools.drivers import forecast
 from uwtools.drivers.driver import Driver
-from uwtools.drivers.forecast import FV3Forecast
+from uwtools.drivers.forecast import FV3Forecast, MPASForecast
 from uwtools.logging import log
 from uwtools.tests.support import compare_files, fixture_path
+from uwtools.utils.file import readable, writable
 
 
-def test_batch_script():
-    expected = """
+class TestForecast:
```

Comment:
I just got around to expanding the diff on this file and saw that you beat me to trying out `pytest` classes! Nice.